mllib/src/test/scala/org/apache/spark/ml/classification/DecisionTreeClassifierSuite.scala

```diff
@@ -57,6 +60,20 @@ class DecisionTreeClassifierSuite extends MLTest with DefaultReadWriteTest {
     categoricalDataPointsForMulticlassForOrderedFeaturesRDD = sc.parallelize(
       OldDecisionTreeSuite.generateCategoricalDataPointsForMulticlassForOrderedFeatures())
       .map(_.asML)
+    linearMulticlassDataset = {
```

Comment:
Is this just used one place? maybe I'm missing it. If so I'd inline it for simplicity